Makefile

```diff
@@ -45,8 +45,13 @@ node_modules: package.json package-lock.json
 
 ifeq ($(CI),false)
 	npm install
+	@echo Install Playwright and its dependencies
+	cd e2e/playwright && npm install
```

Comment:
Could we actually split this into a separate Makefile target so that it's not required when people are just working with the web app? I'm worried about this adding to the time it takes to set up the web app when this isn't used in most cases, and I'd also like to avoid having a side effect on this Makefile target since it's the only one that actually works like Makefiles are supposed to work because the target (`node_modules`) is exactly what's output by running these commands. If we do that, then we can make it so that `make check-types` is the only one that needs us to install the Playwright dependencies

Reply:
Sure, please see https://github.com/mattermost/mattermost-webapp/pull/11468/commits/f60d5a712dd8de22531098c0bfdbf52b9c7e48a3

Reply:
Thanks! The way our Makefile works is a bit of a pet peeve of mine, so I'm glad to see another non-phony target in there 😀